modules/serverless-rn-1-6-0.adoc

```diff
@@ -26,7 +26,7 @@ You can remove the old CRD by entering the following command:
 $ oc delete crd knativeservings.serving.knative.dev
 ----
 ====
-* The *Subscription Update Channel* for new {ServerlessProductName} releases was updated from `techpreview` to `preview-4.3`.
+* The *Subscription Update Channel* for new {ServerlessProductName} releases was updated from `techpreview` to `preview-4.4`.
```

Comment:
@abrennan89, PTAL?

Reply:
@ahardin-rh please revert this - it should stay as `preview-4.3`

Reply:
Thank you!